packages/atomic/src/components/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -271,10 +271,10 @@ export class AtomicRefineModal implements InitializableComponent {
           <hr class="border-neutral"></hr>
           <div class="overflow-auto px-6 flex-grow">
             <div class="adjust-for-scroll-bar">
-              <div class="centered">
+              <main class="centered">
                 {this.renderSort()}
                 {this.renderFilters()}
-              </div>
+              </main>
```

Comment:
As an alternative, [this post](https://stackoverflow.com/questions/3072597/semantically-accurate-html5-element-for-a-modal-dialog) suggests using `aside` elements for modals.